src/main/java/pl/agh/cs/io/Rule.java

```diff
@@ -61,26 +67,24 @@ public void resetTimes() {
     @Override
     public String toString() {
```

Comment:
Nie wiem czy to dobre miejsce, ale może warto by było to jakoś wyciągnąć z tego toStringa, bo to chyba powinna być warstwa prezentacji

Reply:
Chyba wykorzystane jest to do okienka które pokazuje aktualne wykorzystanie, poza tym można by się tego pozbyć

Reply:
Taaak, nie mówię, że to Twoja działka